go.mod

```
@@ -2,7 +2,7 @@ module github.com/hashicorp/consul
 
 go 1.20
```

We could also bump the version here
```suggestion
go 1.22
```

---

I'll try it and make sure all of our tests still pass.